### DISCUSSION TITLE
I'm new to this, what are some of the best resources available to learn Hyperledger fabric?

### DISCUSSION THREAD

**Discussion by alankrit-01** [hyperledger/fabric#3228] [Q&A]
I'm new to this, what are some of the best resources available to learn Hyperledger fabric?

**Comment by arsulegai** [comment]
Hello @amandeep4272 thank you for reaching out.
Few suggestions:
1. Getting started with Hyperledger Fabric's official documentation is an excellent way and probably the right way. For 2.2 release the documentation is at https://hyperledger-fabric.readthedocs.io/en/release-2.2/.
2. There are a couple of learning courses offered by The Linux Foundation. Checkout the developer focussed (application development using Hyperledger Fabric) https://training.linuxfoundation.org/training/hyperledger-fabric-for-developers-lfd272/ and administration focussed (learning DevOps, node operations, permissionning etc) https://training.linuxfoundation.org/training/hyperledger-fabric-administration-lfs272/ courses. Check all other Hyperledger Training Partners https://www.hyperledger.org/learn/hyperledger-training-partners.
3. Hyperledger community compiled resources under the Learning Materials Development Working Group https://wiki.hyperledger.org/display/LMDWG.
4. Do lookout for meet-ups in your city, regional chapters (India has one of them) https://wiki.hyperledger.org/display/HIRC/Hyperledger+India+Regional+Chapter+Home. There are a lot of ways to engage with the community and learn.
